src/njoy21/input/GROUPR/Card2/Iprint.hpp

```diff
@@ -0,0 +1,15 @@
+struct Iprint {
+  using Value_t = int;
+  static std::string name(){ return "iprint"; }
+
+  static std::string description(){
+    return
+        "The iprint argument specifies the verbosity of the printed output. \n"
+        "A value 0 produces the minimum amount of ouput (i.e., quiet) while \n"
+        "a value of 1 produces the maximum.";
```

Comment:
Usually this refers to plotr output, but that doesn't seem to be the case here. What extra output do I get?